### PR TITLE
Use GitHub Actions for ROS1 CI

### PR DESCRIPTION
**Pull request by mm318**
This pull request removes the use of Travis and replaces it with the use of GitHub Actions.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: robot_ws/src/cloudwatch_robot/package.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>cloudwatch_robot</name>
-  <version>1.1.0</version>
```
Comment (raghaprasad) [inline robot_ws/src/cloudwatch_robot/package.xml:4]: This should be a atleast 1.2.0

**Review by raghaprasad — APPROVED** [review]
LGTM for most parts. 
The main workflow seems a bit repetitive to me.
Any ideas on better abstractions?

**Comment by mm318** [comment]
> The main workflow seems a bit repetitive to me.
> Any ideas on better abstractions?

I'll think about making the workspace another dimension under the matrix.

File: robot_ws/src/cloudwatch_robot/package.xml (hunk 1 above)
Comment (mm318) [reply robot_ws/src/cloudwatch_robot/package.xml:4]: Will do.

**Pull request merged**